contrib/for-tests/mount-tester/mt.go

```diff
@@ -152,3 +160,41 @@ func readWriteNewFile(path string) error {
 
 	return readFileContent(path)
 }
+
+func readFileContentInLoop(path string) error {
+	if path == "" {
+		return nil
+	}
+	var content []byte
+	//Expected content "mount-tester new file\n", length (22), retry during 240 second
+	content = TestFileContent(path, 22, 240)
```

Comment:
@dalanlan On thinking about it, why don't you make the duration of the retry a cli parameter?